system/blueprints/config/system.yaml

```diff
@@ -794,6 +794,27 @@ form:
                     validate:
                         type: bool
 
+                assets.precompress:
+                    type: toggle
+                    label: PLUGIN_ADMIN.PRECOMPRESS_ASSET
+                    help: PLUGIN_ADMIN.PRECOMPRESS_ASSET_HELP
+                    highlight: 0
+                    options:
+                        1: PLUGIN_ADMIN.YES
+                        0: PLUGIN_ADMIN.NO
+                    validate:
+                        type: bool
+
+                assets.precompress_level:
+                    type: text
+                    size: x-small
+                    label: PLUGIN_ADMIN.PRECOMPRESS_ASSET_LEVEL
+                    help: PLUGIN_ADMIN.PRECOMPRESS_ASSET_LEVEL_HELP
+                    validate:
+                        type: number
+                        min: 1
+                        max: 9
+
```

Comment:
I see you have provided updates to system.yaml with translations for this field, however, those translations need to be added to the `admin/languages/en.yaml` file as the 'source' language file.